packages/logger/src/base-console-logger-client.ts

```diff
@@ -27,7 +27,12 @@ export abstract class BaseConsoleLoggerClient implements LoggerClient {
     public async setup(baseProperties?: BaseTelemetryProperties): Promise<void> {
         this.baseProperties = baseProperties;
 
-        this.isConsoleLogEnabled = (await this.serviceConfig.getConfigValue('logConfig')).logInConsole;
+        if (this.serviceConfig !== undefined) {
```

Comment:
use lodash isNil

Reply:
also, why will this ever be null? if service config is not found, would we be using the default runtime-config?

Reply:
It would be null in the health-checker script, because we don't have a ServiceConfiguration in that file, and I didn't create one because we would only be using one value anyway to enable/disable console logging. Should we be creating a service config in health-checker?